docs/reference/indices/recovery.asciidoc

```diff
@@ -24,7 +24,7 @@ indices recovery result.
 [source,js]
 --------------------------------------------------
 # create the index
-PUT index1
+PUT index1?include_type_name=true
```

Comment:
Another case of a call where we might ask why the `include_type_name` param is required?